src/components/seo/seoHelmet.js

```diff
@@ -2,10 +2,10 @@ import React from "react"
 import { Helmet } from "react-helmet"
 
 export default function SeoHelmet(props) {
-  const { title, description, canonical } = props
+  const { title, description, canonical, lang = 'en'} = props
```

Comment:
An idea: what if we'd added PropTypes? we'd kill two birds with one stone. You could have type checking and definitions for default/optional props. If we don't need / want type checking, this is ok.

Reply:
Added PropTypes to components.